inspire_disambiguation/core/es/readers.py

```diff
@@ -103,6 +103,31 @@ def get_literature_records_query(signature_block, only_curated):
     return query
 
 
+def get_signature_blocks_query(only_curated):
```

Comment:
do you ever need use this with `False`? if not, better remove the flag and define `get_curated_signature_blocks_query`.